bin/get-all-tasks

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## This script get all tasks for SAPS interacting with Dispatcher backend.
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=$1
+readonly USER_PASSWORD=$2
+readonly DISPATCHER_ADDRESS=$3
+
+ALL_TASKS=$(curl --silent --request GET "${DISPATCHER_ADDRESS}/processings" --header "Content-Type: text/plain"  --header "userEmail: $USER_EMAIL"  --header "userPass: $USER_PASSWORD")
+JSON_LENGTH=$(($(echo $ALL_TASKS | jq 'length') - 1))
+
```

Comment:
I'm wondering if we should list all these properties by default. Or, if we should omit the tag stuff. Maybe, the tag columns (gathering, preproc, proc) should appear when the user gives a --tags option.

what to you think @wesleymonte @thiagoyeds and @raonismaneoto?